app/src/main/admin/components/Touchstones/Demography/DownloadDemographicsPage.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+
+import {DownloadDemographicsContent} from "../../../../shared/components/Demographics/DownloadDemographicsContent";
+import {PageArticle} from "../../../../shared/components/PageWithHeader/PageArticle";
+import {PageProperties} from "../../../../shared/components/PageWithHeader/PageProperties";
+import {AdminPage} from "../../../AdminPage";
+import {downloadDemographicsAdminPageActionCreators} from "../../../actions/pages/downloadDemographicsAdminPageActionCreators";
+import {TouchstoneVersionPageLocationProps} from "../SingleTouchstoneVersion/TouchstoneVersionPage";
+
+class DownloadDemographicsAdminPageComponent extends React.Component<PageProperties<TouchstoneVersionPageLocationProps>> {
+
+    render(): JSX.Element {
+        return <PageArticle title={this.props.title}>
```

Comment:
The difference between this component and `DownloadDemographicsContribPageComponent` is just the formatting of the title. In the contrib component we mention the current responsibility in the title.